lib/mw/log.js

```diff
@@ -38,7 +67,7 @@ exports.attach = function(app, options) {
     message.unshift(((Date.now() - this.started) / 1000) + '');
     message.unshift(event);
 
-    console.log(message.join('\t'));
+    logger.info(message.join('\t'));
```

Comment:
If we're going to add Winston, we should actually use the interface instead of the lazy TSV format I had: https://github.com/winstonjs/winston#logging-with-metadata